modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,75 @@
+// TODO The Envoy numbers goes down in 1.9, check for the latest data with next version of Istio.
+The data plane components, the Envoy proxies, handle data flowing through the system. The control plane component, Istiod, configures the data plane. The data plane and control plane have distinct performance concerns.
+
+== Control plane performance
+
+Istiod configures sidecar proxies based on user authored configuration files and the current state of the system.
+In a Kubernetes environment, Custom Resource Definitions (CRDs) and deployments constitute the configuration and state of the system.
+The Istio configuration objects like gateways and virtual services, provide the user-authored configuration.
+To produce the configuration for the proxies, Istiod processes the combined configuration and system state from the Kubernetes environment and the user-authored configuration.
+
+The control plane supports thousands of services, spread across thousands of pods with a similar number of user authored virtual services and other configuration objects.
+Istiod's CPU and memory requirements scale with the number of configurations and possible system states.
+The CPU consumption scales with the following factors:
+
+* The rate of deployment changes.
+* The rate of configuration changes.
+* The number of proxies connecting to Istiod.
+
+However this part is inherently horizontally scalable.
+
+//Do we support namespace isolation?  When namespace isolation is enabled, a single Istiod instance can support 1000 services, 2000 sidecars with 1 vCPU and 1.5 GB of memory.
+//You can increase the number of Istiod instances to reduce the amount of time it takes for the configuration to reach all proxies.
+
+== Data plane performance
+
+Data plane performance depends on many factors, for example:
+
+* Number of client connections
+* Target request rate
+* Request size and response size
+* Number of proxy worker threads
+* Protocol
+* CPU cores
+* Number and types of proxy filters, specifically telemetry v2 related filters.
+
+The latency, throughput, and the proxies' CPU and memory consumption are measured as a function of said factors.
+
+=== CPU and memory
+
+Since the sidecar proxy performs additional work on the data path, it consumes CPU and memory. As of Istio 1.1, a proxy consumes about 0.6 vCPU per 1000 requests per second.
+//TODO As of Istio 1.7, a proxy consumes about 0.5 vCPU per 1000 requests per second.
+
+The memory consumption of the proxy depends on the total configuration state the proxy holds.
+A large number of listeners, clusters, and routes can increase memory usage.
+//Istio 1.1 introduced namespace isolation to limit the scope of the configuration sent to a proxy. In a large namespace, the proxy consumes approximately 50 MB of memory.
+
+Since the proxy normally doesn't buffer the data passing through, request rate doesn't affect the memory consumption.
+
+=== Latency for Istio 1.6.8
```

Comment:
@JStickler Can we change this to just Latency for the same reasons we talked about?